lux-cli/src/shell.rs

```diff
@@ -0,0 +1,72 @@
+use clap::Args;
+use eyre::Result;
+use lux_lib::config::Config;
+
+use std::env;
+use tokio::process::Command;
+
+use lux_lib::{path::Paths, project::Project};
+
+use std::process::Stdio;
+
+#[derive(Args)]
+pub struct Shell {
+    #[arg(long)]
+    test: bool,
+    #[arg(long)]
+    build: bool,
+}
+
+pub async fn shell(data: Shell, config: Config) -> Result<()> {
+    let project =
+        Project::current()?.ok_or_else(|| eyre::eyre!("Not in a Lux project directory"))?;
+    let project_tree = project.tree(&config)?;
```

Comment:
todo: `lx shell` should also work if there's no project in the cwd.
Use `utils::current_project_or_user_tree` instead.